### PR TITLE
My solution2

### PR DESCRIPTION
**Pull request by Adrian-Mag**

### CODE REVIEW

File: prototype.py [hunk 1]
```diff
@@ -1,3 +1,5 @@
+import matplotlib
```
Comment (synapsidfan) [inline prototype.py:1]: This file should be removed from the repository.

File: pkmodel/model.py [hunk 2]
```diff
@@ -4,14 +4,78 @@
 
 class Model:
     """A Pharmokinetic (PK) model
+    -------
+    INPUTS: dictionary describing the model
```
Comment (sscivier) [inline pkmodel/model.py:8]: Put an underline under inputs, will make the docs look better

File: pkmodel/model.py [hunk 3]
```diff
@@ -4,14 +4,78 @@
...
+    !! The peripheral compartiments must be added starting from
+    1 and increasing in increments of 1 up to the maximum number
+    peripheral compartiments used !!
 
-    value: numeric, optional
-        an example paramter
+    !! At a minimum the model must have a main compartiemnt with
+    CL and Vc given and a name !!
+    ----------
+    OUTPUTS: model object that holds all the properties as attributes
```
Comment (sscivier) [inline pkmodel/model.py:31]: move the 'model object that ...' underneath the underline on outputs - again to make the docs look nicer

**Review by synapsidfan — CHANGES_REQUESTED** [review]
Once the prototype.py file has been removed, this should be good to merge

File: pkmodel/solution.py [hunk 4]
```diff
@@ -1,17 +1,166 @@
...
+import scipy.integrate
+import numpy as np
+import matplotlib
+import matplotlib.pylab as plt
+matplotlib.use('TkAgg')
+
 
 class Solution:
-    """A Pharmokinetic (PK) model solution
+    """A Pharmokinetic (PK) model
```
Comment (sscivier) [inline pkmodel/solution.py:12]: Where did 'solution' go?

**Review by sscivier — CHANGES_REQUESTED** [review]
- More commenting needed in general to explain the 'why'
- Re-format docstrings according to 'Parameters', 'Returns'

File: pkmodel/model.py [hunk 5]
```diff
@@ -32,28 +32,33 @@ class Model:
...
         'Vp2': Vp1 <float>, [mL] (if two peripheral used)
+
         'Qp3': Qp1 <float>, [mL/h] (if three peripheral used)
+
         'Vp3': Vp1 <float>, [mL] (if three peripheral used)
         ... etc
     }
 
-    ** The peripheral compartiments must be added starting from
+    **The peripheral compartiments must be added starting from
```
Comment (synapsidfan) [inline pkmodel/model.py:56]: Compartments is misspelled here

File: pkmodel/model.py [hunk 6]
```diff
@@ -32,28 +32,33 @@ class Model:
...
     1 and increasing in increments of 1 up to the maximum number
-    peripheral compartiments used **
-
-    ** At a minimum, the model must have a main compartiemnt with
-    CL and Vc given, and a name **
+    peripheral compartiments used**
```
Comment (synapsidfan) [inline pkmodel/model.py:58]: Spelling of compartments should also be fixed here

File: pkmodel/model.py [hunk 7]
```diff
@@ -32,28 +32,33 @@ class Model:
...
 
-    OUTPUTS:
-    -----------------------------------------------------------------
-    model object that holds all the properties as attributes
+    **At a minimum, the model must have a main compartiemnt with
```
Comment (synapsidfan) [inline pkmodel/model.py:60]: Spelling issue here too

File: pkmodel/solution.py [hunk 8]
```diff
@@ -14,8 +14,9 @@ class Solution:
     INPUTS
     ------------
     model: object
-
-    protocol object
+        see Model class
+    protocol: object
+        see Protocol class
 
     This class creates solution objects that contain the time interval,
```
Comment (synapsidfan) [inline pkmodel/solution.py:21]: Perhaps say "one or more solution objects" so as to not imply that you will always get more than one?

**Review by synapsidfan — CHANGES_REQUESTED** [review]
Just a few minor spelling issues and clarity fixes are needed.

**Pull request closed without merging**